pkg/backingstore/backingstore.go

```diff
@@ -427,7 +429,28 @@ func RunDelete(cmd *cobra.Command, args []string) {
 	backStore.Namespace = options.Namespace
 	backStore.Spec = nbv1.BackingStoreSpec{}
 
-	if !util.KubeDelete(backStore) {
+	nbClient := system.GetNBClient()
+
+	continueDelete := true
+	poolinfo, err := nbClient.ReadPoolAPI(nb.ReadPoolParams{Name: backStore.Name})
+	if err != nil {
+		if rpcErr, isRPCErr := err.(*nb.RPCError); !isRPCErr || rpcErr.RPCCode != "NO_SUCH_POOL" {
+			log.Fatalf(`❌ Failed to read BackingStore info: %s`, err)
+			continueDelete = false
```

Comment:
log.Fatalf() exits the process after logging, so no need for the code after it, so continueDelete is unneeded

Reply:
fixed